vulnerabilities/templates/vulnerabilities.html

```diff
@@ -77,5 +87,13 @@
       {% endif %}
     </section>
 {% endif %}
-
-{% endblock %}
+<script>
+    function changeItemsPerPage(pageSize) {
+        var urlParams = new URLSearchParams(window.location.search);
+        urlParams.set('page_size', pageSize);
+        window.location.search = urlParams.toString();
+    }
+    
+</script>
```

Comment:
Duplicated code.

Reply:
Now I have made a `pagination.js` file in `static/js ` and certainly used `<script src="{% static 'js/pagination.js' %}"></script>` to avoid duplication of code.